Automattic/wp-calypso
# Checkout: Remove useFetchProductsIfNotLoaded

## Proposed Changes

In https://github.com/Automattic/wp-calypso/pull/63292 we modified checkout so that it no longer requires a list of products to add products to the cart. However, there's still a hook, `useFetchProductsIfNotLoaded`, which is run by checkout to fetch those products. That hook is removed by this PR.

## Testing Instructions

The hook was added to help add products from the URL in https://github.com/Automattic/wp-calypso/pull/40725 so we can verify that still works:

- Visit checkout with a product in the URL like `/checkout/personal`.
- Verify that the product is added successfully.
